1.0/docs/devguide/properties.md

```diff
@@ -524,16 +543,29 @@ Example:
 
     });
 
-**Array mutation methods.** Observing changes to arrays is dependent on the change to the array
-being made through one of the [array mutation methods](#array-mutation) provided
-on Polymer elements, which provides the required notification to elements with
-registered interest.
-{: .alert .alert-info }
 
-When you specify a wildcard path on an array, the observer is for both splices as
-well as array element sub-property changes.  So the  observer in the
-following example will be called for all additions, removals, and deep changes
-that occur in the array:
+#### Wildcard observers
+
+Wildcard observers watch for both [splices](#splice-observers) and for 
```

Comment:
This section duplicates the #deep-observation section above. We need to re-think this structure.

Perhaps it would be clearer if we reverse the ordering of the deep path and array observation sections, so we have an ordering like this:

-   Observe sub-property changes
    No changes here.
  
-   Observe array mutations
    Describe the special ".splices" path for observing array mutations. Mention that if they're interested in both mutations _and_ property changes to array items, they need  a deep observer.
    Describe the change record format for ".splices" changes.
-   Observe deep sub-property changes
    Here, you can use the `changeRecord.path` property to figure out whether you're seeing a property change, sub-property change, or an array mutation.